api/services/Flairs.js

```diff
@@ -189,6 +189,26 @@ exports.flairCheck = function (ptrades, svex) {
   }
   var ptradesFlair = "(([0-9]{4}-){2}[0-9]{4})(, (([0-9]{4}-){2}[0-9]{4}))* \\|\\| ([^,|(]*( \\((X|Y|ΩR|αS)(, (X|Y|ΩR|αS))*\\))?)(, ([^,|(]*( \\((X|Y|ΩR|αS)(, (X|Y|ΩR|αS))*\\))?))*";
   var svExFlair = ptradesFlair + " \\|\\| ([0-9]{4}|XXXX)(, (([0-9]{4})|XXXX))*";
+  var tradesParts = ptrades.split("||");
+  var svexParts = svex.split("||");
```

Comment:
I think we disable that in the front-end and the regex later on will obviously fail. I can, however, add another check specifically if you want.